onnx/test/version_converter_test.py

```diff
@@ -646,6 +647,34 @@ def test_constant_9_8(self):  # type: () -> None
         assert converted_model.graph.output[0].type.tensor_type.elem_type == data_type
         assert converted_model.opset_import[0].version == to_opset
 
+    # Test Constant Adapter: 9 -> 8
+    def test_constant_9_8_polish(self):  # type: () -> None
+        from_opset = 9
+        to_opset = 8
+        data_type = TensorProto.UINT64
```

Comment:
Can we also add tests for UINT64, and FLOAT/DOUBLE cases

Reply:
We have a test for UINT64.